build_python_wheel.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+
+
+function print_help {
+  echo "Configures and builds a specified target."
+  echo
+  echo "Usage: ./build.sh [build options] [configure options]"
+  echo
+  echo "Common Options:"
+  echo "  --target-dir, -t                 The target directory to install artifact to."
+  echo "                                   default: `pwd`/targets."
+  echo
+  echo "  --release                        Build in release mode."
```

Comment:
It doesn't look like debug or release is actually getting used any where.

Reply:
by default is release. Just add `--debug` should be fine.